src/rabbit-event-bus/index.ts

```diff
@@ -20,21 +20,29 @@ export interface RabbitEventBusConnectionOptions {
  * @implements {EventBus}
  */
 export default class RabbitEventBus implements EventBus, ConnectionOwner {
-    private connector: Option<AMQPConnector> = None;
+    private _connector: Option<AMQPConnector> = None;
     private connection: ConnectionObserver;
-    private eventDefinitions: string[];
+    private eventToHandle: EventType[];
     private serviceName = 'unknown-service';
     private url = '';
     private queue: InternalMessageQueue;
-    private subscriptions: Array<Subscription<unknown & object>> = [];
+    private _subscriptions: Array<Subscription<unknown & object>> = [];
 
     public constructor(connectionOpts: RabbitEventBusConnectionOptions) {
         this.url = connectionOpts.url;
     }
 
+    public get connector(): Option<AMQPConnector> {
+        return this._connector;
+    }
+
+    public get subscriptions(): Array<Subscription<unknown & object>> {
+        return this._subscriptions;
+    }
+
     // Can the params here be moved to the constructor?
```

Comment:
Commented on the mock the same thing. Can we get rid of `register` completely in favour of constructor?

Reply:
@Cooryd Done